DEVELOPER_GUIDE.md

```diff
@@ -275,6 +279,13 @@ This method can also be used to develop against the [full distribution of OpenSe
 
 _This step is only mandatory if you have the [`security` plugin](https://github.com/opensearch-project/security) installed on your OpenSearch cluster with https/authentication enabled._
 
+> Running `yarn opensearch snapshot -P <security_plugin_url>` will not work since the plugin needs some additional configuration to setup correctly. To configure it correctly, here are the steps:
+>
+> 1. Run `yarn opensearch snapshot -P <security_plugin_url>`. This will fail to run and complain about missing certificates. Ignore that since what we want is for the build artifacts
+> 2. Run `export initialAdminPassword=<inital admin password>` since its needed by the configuration script
+> 3. Run the config script in the plugin directory of opensearch. `bash .opensearch/<version>/plugins/opensearch-security/tools/install_demo_configuration.sh`
```

Comment:
@manasvinibs do you think it's worth to consider provide an optional execute setup script or even "with security enabled that runs this default'

Reply:
That would be even better. A one line command would be ideal

Reply:
Yeah I agree! An additional flag to enable security which takes care of `step 2` and `step 3` will be neat.

Reply:
Do we want a separate PR for the additional flag?